source/security/index.html.md.erb

```diff
@@ -68,7 +68,12 @@ If you think you’re receiving fraudulent payments, you can:
 - [delay capture of payments](/delayed_capture/#delay-taking-a-payment) if you need time to do your own anti-fraud checks
 - check your payment service provider's (PSP's) security settings, or [contact us](/support_contact_and_more_information/#support) if your PSP is Stripe
 
-If your PSP is Worldpay, you can also set up your account to [make 'risk management' fraud checks](http://support.worldpay.com/support/kb/gg/merchantadmininterface/Merchant%20Interface%20Guide.htm#5risk/risk_management.htm%3FTocPath%3DRisk%7C_____2). If you do this, you must [contact us](https://docs.payments.service.gov.uk/support_contact_and_more_information/) and ask us to send your user's IP address to Worldpay during each payment, or your payments may stop working.
+You can also set up your account to make 'risk management' fraud checks if your PSP is:
+
+- [Worldpay](http://support.worldpay.com/support/kb/gg/merchantadmininterface/Merchant%20Interface%20Guide.htm#5risk/risk_management.htm%3FTocPath%3DRisk%7C_____2)
+- [SmartPay](https://help.barclaycard.co.uk/smartpaysecure/setting-up-product/configure-fraud-checks)
```

Comment:
For our Smartpay integration, I think this is the risk management info https://docs.adyen.com/risk-management (from team manual - `Smartpay is Barclaycard’s rebadged version of Adyen`). Smartpay secure is probably a different version/product

Reply:
Ah good spot, thanks - updated in https://github.com/alphagov/pay-tech-docs/pull/543/commits/45cac3d3b59d3456e22e06159df0dfe2543dfa03.